lib/csv.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs'
+import { parse } from 'csv-parse/sync'
```

Comment:
Is a CSV parser necessary considering that the file only has 1 column and could just be read line by line?

Reply:
This is a fair point, might do a quick refactor to just make it a read-from-txt file type deal